isso/js/app/dom.js

```diff
@@ -32,8 +32,16 @@ define(function() {
         /**
          * Shortcut for `Element.addEventListener`, prevents default event
          * by default, set :param prevents: to `false` to change that behavior.
+         * You can also provide an array of types, to listen on multiple events
          */
         this.on = function(type, listener, prevent) {
+            if (Array.isArray(type)) {
+                var that = this;
+                type.forEach(function (type) {
```

Comment:
Shouldn't a different var be used for the internally-scoped `type`? Even if just for clarity...